contribute/create-pull-request.md

```diff
@@ -43,6 +43,10 @@ Pull requests for Redux contributions must:
 - Not contain code that mutates state in reducers or thunks.
 - Not contain code that accesses the reducers state slice directly. Instead, the code should use state selectors to access state.
 
+Pull requests that add/modify unit test (Jest) must:
+
+- Avoid adding snapshots tests, we are removing current ones incrementally
```

Comment:
Why "Avoid adding" and not "Don't add"? Are we looking to completely remove snapshot tests, or are there still valid use cases?

Reply:
We're currently migrating from Enzyme to RTL one test at the time.

Reply:
I'm not sure I understand how this answers the question? If we don't allow new snapshot tests at all, this should say "Don't add" rather than "Avoid". The latter implies that it's ok to add snapshot tests in some circumstances.

Reply:
yes, it can be confusing the use of 'avoid adding', I will change it to 'Don't add', because we don't want new snapshots :)